tests/test_file_open_gui.py

```diff
@@ -0,0 +1,449 @@
+    assert app_dialog.root.captions["My Tasks"].exists(), "Not the right breadcrumb"
+
+    # Make sure the all tabs are showing up.
+    assert app_dialog.root.tabs["My Tasks"].exists(), "My Tasks tab is missing"
+    assert app_dialog.root.tabs["Assets"].exists(), "Assets tab is missing"
+    assert app_dialog.root.tabs["Shots"].exists(), "Shots tab is missing"
+    assert app_dialog.root.tabs["All"].exists(), "All tab is missing"
+    assert app_dialog.root.tabs["Working"].exists(), "Working tab is missing"
+    assert app_dialog.root.tabs["Publishes"].exists(), "Publishes tab is missing"
+
+    # Make sure all buttons are showing up
+    assert app_dialog.root.buttons[
+        "+ New Task"
+    ].exists(), "+ New Task button is missing"
+    assert app_dialog.root.buttons[
+        "+ New File"
+    ].exists(), "+ New File button is missing"
+    assert app_dialog.root.buttons["Cancel"].exists(), "Cancel button is missing"
+    assert app_dialog.root.buttons["Open"].exists(), "Open button is missing"
+
+    # Make sure all test fields are showing up
+    assert app_dialog.root.textfields[
+        0
+    ].exists(), "Search All Files text field is missing"
+    assert app_dialog.root.textfields[
+        1
+    ].exists(), "Search My Tasks text field is missing"
+
+    # Make sure all checkboxes are showing up
+    assert app_dialog.root.checkboxes[
+        "All Versions"
+    ].exists(), "All Versions checkbox is missing"
+
+
+def test_assets_tab(app_dialog):
+    # Select the Assets tab
+    app_dialog.root.tabs["Assets"].mouseClick()
+
+    # Make sure all expected UI is showing up
+    assert app_dialog.root.captions[
+        "Filter by Pipeline Step"
+    ].exists(), "Pipeline Step filters are missing"
+    assert app_dialog.root.buttons[
+        "Select All"
+    ].exists(), "Pipeline Step filters Select All button is missing"
+    assert app_dialog.root.buttons[
+        "Select None"
+    ].exists(), "Pipeline Step filters Select None button is missing"
+    assert app_dialog.root.checkboxes[
+        "My Tasks Only"
+    ].exists(), "My Tasks Only checkbox is missing"
+    assert app_dialog.root.buttons[
+        "+ New Task"
+    ].exists(), "+ New Task button is missing"
+    assert app_dialog.root.textfields[1].exists(), "Search Assets text field is missing"
+
+    # Got to the model task and validate breadcrumb
+    app_dialog.root.outlineitems["Character"].waitExist(timeout=30)
+    app_dialog.root.outlineitems["Character"].mouseDoubleClick()
+    app_dialog.root.outlineitems["AssetAutomation"].waitExist(timeout=30)
+    app_dialog.root.outlineitems["AssetAutomation"].mouseDoubleClick()
+    app_dialog.root.outlineitems["Model"].waitExist(timeout=30)
+
+    # Validate content dialog
+    assert app_dialog.root.cells[
+        "AssetAutomation"
+    ].exists(), "AssetAutomation is missing in content dialog"
+    # These tests are failing on Azure but succeed locally. Need more investigation
```

Comment:
instead of commenting them out, you could do

`if "CI" not in os.environ:`

and put your code inside the if.

Reply:
This way it would keep running locally.

Reply:
oh nice!